Makefile

```diff
@@ -112,7 +112,7 @@ v8:
 	$(MAKE) -C deps/v8 $(V8_ARCH) $(V8_BUILD_OPTIONS)
 
 test: | cctest  # Depends on 'all'.
-	$(PYTHON) tools/test.py --mode=release message parallel sequential -J
+	$(PYTHON) tools/test.py --mode=release doctool message parallel sequential -J
```

Comment:
Very minor, but for consistency I'd suggest adding the new directory last here, like you did with `test-ci` below.  Same goes for `vcbuild.bat`.